### PR TITLE
OPR-1839: adding cheerio, pdf-parse

### PR DESCRIPTION
**Pull request by dbo**
@chanakyabhardwajj please

### CODE REVIEW

File: npm/cheerio/1.0.0/index.n4jsd [hunk 1]
```diff
@@ -0,0 +1,43 @@
+/**
+ * @see https://github.com/cheeriojs/cheerio
+ * 
+ * TODO incomplete
```
Comment (chanakyabhardwajj) [inline npm/cheerio/1.0.0/index.n4jsd]: Please mention the package version and elaborate what is the `TODO` here.

File: npm/pdf-parse/1.0.8/index.n4jsd [hunk 2]
```diff
@@ -0,0 +1,22 @@
...
+    public metadata: Object+; 
+    // PDF.js version
+    // check https://mozilla.github.io/pdf.js/getting_started/
+    public version: string;
+    // PDF text
+    public text: string;
+}
+
+/**
+ * @see https://www.npmjs.com/package/pdf-parse
```
Comment (chanakyabhardwajj) [inline npm/pdf-parse/1.0.8/index.n4jsd:18]: Please mention the package version.

**Review by chanakyabhardwajj — APPROVED** [review]

File: npm/cheerio/1.0.0/index.n4jsd (hunk 1 above)
Comment (dbo) [reply npm/cheerio/1.0.0/index.n4jsd]: removed the TODO since all n4jsd lack accuracy in detail

File: npm/pdf-parse/1.0.8/index.n4jsd (hunk 2 above)
Comment (dbo) [reply npm/pdf-parse/1.0.8/index.n4jsd:18]: is already in the manifest

**Pull request merged**